### DISCUSSION TITLE
How to update domain registrant-contact

### DISCUSSION THREAD

**Discussion by jaumann** [aws/aws-cli#7002] [Q&A] [answered]
Hi all! Hopefully someone out there can help me because I can't seem to find the answer in any of the documentation or code. I'm attempting to standardize the domain contact info for our domains registered in route53 after my company's recent name change. I can programatically change the admin and tech contact information easily enough using `aws route53domains update-domain-contact --domain-name <DOMAIN> --admin-contact <CONTACT INFO> --tech-contact <CONTACT INFO>`.

If I attempt to change the `--registrant-contact` (our company name is in the first name of the contact, so that is included in what's being changed), I get back an error stating ```An error occurred (InvalidInput) when calling the UpdateDomainContact operation: Changing owner's first or last name is a change of ownership.```. I can't seem to figure out how to allow ownership contact changes via the aws-cli. Is there any way to do this? Thanks!

**Comment by tim-finnigan** [comment, marked as answer]
Hi @jaumann thanks for reaching out. I'm not aware of a route53domains API or CLI command for updating the ownership of a domain. Based on this documentation it seems like changing the ownership is a somewhat involved process: https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-update-contacts.html.
